apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-customization.tsx

```diff
@@ -62,7 +62,7 @@ interface SignInMethodCustomizationPropsInterface extends SBACInterface<FeatureC
     /**
      * Callback to trigger IDP create wizard.
      */
-    onIDPCreateWizardTrigger: (type: string, cb: () => void) => void;
+    onIDPCreateWizardTrigger: (type: string, cb: () => void, template?: any) => void;
```

Comment:
Let's get rid of `any` in a different PR.

```suggestion
    onIDPCreateWizardTrigger: (type: string, cb: () => void, template?: IdentityProviderTemplateInterface | IdentityProviderTemplateGroupInterface) => void;
```

Reply:
+1 Mentioning the mapping issue here for reference: https://github.com/wso2-enterprise/asgardeo-product/issues/3878